internal/pkg/agent/application/application.go

```diff
@@ -111,10 +116,18 @@ func New(
 	}
 	monitor := monitoring.New(isMonitoringSupported, cfg.Settings.DownloadConfig.OS(), cfg.Settings.MonitoringConfig, agentInfo)
 
+	controlAddress := control.Address()
+	log.Debugf("Control address: %s", controlAddress)
```

Comment:
We already have a info level message that logs the control address.

https://github.com/elastic/elastic-agent/blob/main/pkg/control/v2/server/server.go#L88